openroad_interface/length_calculations.py

```diff
@@ -0,0 +1,64 @@
+import copy
+from var import directory
+
+
+def length_calculations(units: float) -> dict: 
+    '''
+    calculates lengths of each net using the def file
+
+    params: 
+        units: to convert length into correct units
+    returns: 
+        length_dict: length with macro ID attribution
+    '''
+
+    def_file = "results/final_generated-tcl.def"
```

Comment:
this shouldn't be hardcoded. IT should instead be a parameter with a default value.

`def length_calculations(units: float, def_file: str = "results/final_generated-tcl.def") -> dict`

Reply:
make this change in all the functions